subcommand/controller/command.go

```diff
@@ -111,28 +111,44 @@ func (c *Command) Run(args []string) int {
 		return 1
 	}
 
-	if err = (&controllers.ServiceDefaultsReconciler{
-		Client:                     mgr.GetClient(),
-		Log:                        ctrl.Log.WithName("controllers").WithName("ServiceDefaults"),
-		Scheme:                     mgr.GetScheme(),
+	configEntryReconciler := &controllers.ConfigEntryController{
 		ConsulClient:               consulClient,
 		EnableConsulNamespaces:     c.flagEnableNamespaces,
 		ConsulDestinationNamespace: c.flagConsulDestinationNamespace,
 		EnableNSMirroring:          c.flagEnableNSMirroring,
 		NSMirroringPrefix:          c.flagNSMirroringPrefix,
 		CrossNSACLPolicy:           c.flagCrossNSACLPolicy,
+	}
+	if err = (&controllers.ServiceDefaultsController{
+		ConfigEntryController: configEntryReconciler,
+		Client:                mgr.GetClient(),
+		Log:                   ctrl.Log.WithName("controllers").WithName("ServiceDefaults"),
```

Comment:
***continues whispering*** _omg those constants would be so handy in these places we use `"ServiceDefaults"` and `"ServiceResolver"`_ 😉

Reply:
Did you want that for this PR? I thought we were gonna get it in on the next one.

Reply:
we can do it in the next one. this aint no blocker